tests/integration/targets/ntnx_clusters_and_hosts_v2/tasks/cluster_crud.yml

```diff
@@ -134,6 +134,9 @@
       http_proxy_white_list:
         - target_type: IPV4_NETWORK_MASK
           target: test
+    categories:
```

Comment:
@abhinavbansal29
Lets update the create cluster test as well, as create spec wont test actual cluster create.
Do we have cluster categories update as well ? Please test.